share/doc/homebrew/El_Capitan_and_Homebrew.md

```diff
@@ -4,7 +4,7 @@ Part of the OS X 10.11/El Capitan changes is something called [System Integrity
 
 SIP prevents you from writing to many system directories such as `/usr`, `/System` & `/bin`, regardless of whether or not you are root. The Apple keynote is [here](https://developer.apple.com/videos/wwdc/2015/?id=706) if you'd like to learn more.
 
-One of the implications of SIP is that you cannot simply create `/usr/local` if you have removed it. However, as noted in the keynote, Apple is leaving `/usr/local` open for developers to use, so Homebrew can still be used as expected.
+One of the implications of SIP was that you could not simply create `/usr/local` if you had removed it. This issue was fixed with the OS X 10.11.0 release. However, as noted in the keynote, Apple is leaving `/usr/local` open for developers to use, so Homebrew can still be used as expected.
```

Comment:
Works for me. I don't know if the change was snuck in as part of the final release or as a security tweak afterwards, but the difference isn't important to people who will read this document.